integrations/agntcy-dir/Taskfile.yml

```
@@ -12,8 +12,8 @@ vars:
   ## Image config
   IMAGE_REPO: '{{ .IMAGE_REPO | default "ghcr.io/agntcy" }}'
 
-  DIRECTORY_IMAGE_TAG: '{{ .DIRECTORY_IMAGE_TAG | default "v0.4.0" }}'
-  DIRECTORY_CHART_TAG: '{{ .DIRECTORY_CHART_TAG | default "v0.4.0" }}'
+  DIRECTORY_IMAGE_TAG: '{{ .DIRECTORY_IMAGE_TAG | default "v0.5.6" }}'
```

```suggestion
  DIRECTORY_VERSION: 'v0.5.6'
  DIRECTORY_IMAGE_TAG: '{{ .DIRECTORY_IMAGE_TAG | default .DIRECTORY_VERSION }}'
```
and replace the other hardcoded versions below